js-exercises/sum-all/sumAll.js

```diff
@@ -0,0 +1,44 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const areNumbers = (args) => args.every(isNumber);
+
+function* getRange(start, end) {
```

Comment:
A for loop is preferred since it is less likely to make an infinite loop there.

Reply:
Also, start can have a good default value such as 0.
```suggestion
function* getRange(start = 0, end) {
```